blockchain/scriptval.go

```diff
@@ -170,13 +172,11 @@ func (v *txValidator) Validate(items []*txValidateItem) error {
 		case err := <-v.resultChan:
 			processedItems++
 			if err != nil {
-				close(v.quitChan)
```

Comment:
This would no longer stop validating the other scripts when one of them errored which is the intention.  It would need to call `cancel()` here.